test/scoped-dir-2/module-in-scoped-dir-2-whose-load-side-effects.js

```diff
@@ -0,0 +1 @@
+global.moduleScopedDir2LoadSideEffect++
```

Comment:
I'd name this `moduleScopedDir2LoadAttempts`

Reply:
I think I'll keep it if you don't mind, since I think the global is indeed the `side effect` of the load.